assets/queries/cloudFormation/api_gateway_v2_stage_not_access_logging_settings/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "api_gateway_v2_stage_not_access_logging_settings",
+  "queryName": "API Gateway V2 Stage Not Access Logging Settings",
```

Comment:
A more suitable name would be "API Gateway V2 Stage Access Logging Settings Not Defined"